api/types/container/nat_aliases.go

```diff
@@ -14,11 +18,12 @@ type PortRangeProto = nat.Port
 type PortSet = nat.PortSet
 
 // PortBinding represents a binding between a Host IP address and a [HostPort].
-//
-// It is currently an alias for [nat.PortBinding] but may become a concrete type in a future release.
-type PortBinding = nat.PortBinding
+type PortBinding struct {
+	// HostIP is the host IP Address
+	HostIP netip.Addr `json:"HostIp"`
+	// HostPort is the host port number
+	HostPort string
+}
```

Comment:
I'm a bit lost in all the port-related types we have; so could use the extra eyes from @akerouanton and @robmry here - to verify if we don't have fields that we use / need to use as "optional" here.